File: server/api/app/masterdata.py
import sqlite3

def dict_factory(cursor, row):
    d = {}
    for idx, col in enumerate(cursor.description):
        d[col[0]] = row[idx]
    return d



class m_game_param:
    
    key: str
    
    value: int
    
    @classmethod
    def select(cls, cond: str='', param: str=''):
        con = sqlite3.connect("/var/masterdata.db")
        con.row_factory = dict_factory
        cur = con.cursor()
        for row in cur.execute(f'SELECT * FROM m_game_param {cond}', param):
            r = cls()
            for k, v in row.items():
                r.__setattr__(k, v)
            yield r
        con.close()


class m_home_image:
    
    name: str
    
    asset_path: str
    
    @classmethod
    def select(cls, cond: str='', param: str=''):
        con = sqlite3.connect("/var/masterdata.db")
        con.row_factory = dict_factory
        cur = con.cursor()
        for row in cur.execute(f'SELECT * FROM m_home_image {cond}', param):
            r = cls()
            for k, v in row.items():
                r.__setattr__(k, v)
            yield r
        con.close()


class asset:
    
    path: str
    
    url: str
    
    @classmethod
    def select(cls, cond: str='', param: str=''):
        con = sqlite3.connect("/var/masterdata.db")
        con.row_factory = dict_factory
        cur = con.cursor()
        for row in cur.execute(f'SELECT * FROM asset {cond}', param):
            r = cls()
            for k, v in row.items():
                r.__setattr__(k, v)
            yield r
        con.close()

File: client/Assets/Scripts/Common/MasterData.cs
using System.Collections.Generic;
using SQLite4Unity3d;

namespace MasterData
{
    
    public class m_game_param
    {
        
        public string key { get; set; }
        
        public int value { get; set; }
        

        public static IEnumerable<m_game_param> Select(string cond="", params object[] args)
        {
            return (new SQLiteConnection(Config.Instance.MasterDataPath, SQLiteOpenFlags.ReadOnly))
                .Query<m_game_param>($"SELECT * FROM m_game_param {cond}", args);
        }
    }
    
    public class m_home_asset
    {
        
        public string name { get; set; }
        
        public string asset_path { get; set; }
        

        public static IEnumerable<m_home_asset> Select(string cond="", params object[] args)
        {
            return (new SQLiteConnection(Config.Instance.MasterDataPath, SQLiteOpenFlags.ReadOnly))
                .Query<m_home_asset>($"SELECT * FROM m_home_asset {cond}", args);
        }
    }
    
    public class asset
    {
        
        public string path { get; set; }
        
        public string url { get; set; }
        

        public static IEnumerable<asset> Select(string cond="", params object[] args)
        {
            return (new SQLiteConnection(Config.Instance.MasterDataPath, SQLiteOpenFlags.ReadOnly))
                .Query<asset>($"SELECT * FROM asset {cond}", args);
        }
    }
    
}